ckpts/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
We do not need this file here. One .gitignore on the project level directory should be enough.

Reply:
This is a trick to keep the folder to not being ignored when pushing the code to GitHub and this ensures that the content of this folder is ignored.

So if someone trains a model and stores their weights here, you don't want them to mistakenly push the weight files also. However, we still want to have the structure exist on your local machine vs. what exists on the Github repo and I have stated before, GitHub removes folders that don't contain anything.

Reply:
tbh, it is a neat trick.